src/bios/f-init32/src/interrupts/handlers.rs

```diff
@@ -0,0 +1,76 @@
+use alloc::boxed::Box;
+use alloc::format;
+use core::arch::asm;
+use core::mem::transmute;
+use f_macros::{interrupt, interrupt_default};
+use flib::debug::debug::print_str;
+use flib::int::scheduler::{IntScheduler, ScheduledAction};
+use flib::{print, scheduler_ref};
+
+/// This module defines every interrupts referenced by the IDT
+/// It provides several utilities to define interrupts.
+/// To define a simple interrupt, precede your fn definiton with the proc
+/// macro #\[interrupt]
+/// # Example
+/// ```
+/// #[interrupt]
+/// pub fn _int0x00() {
+///     do_something()
+/// }
+/// ```
```

Comment:
```suggestion
///
/// # Examples:
///
/// ```
/// #[interrupt]
/// pub fn _int0x00() {
///     do_something()
/// }
/// ```
///
```